examples/feature-api-functions/ponder.schema.ts

```diff
@@ -1,17 +1,17 @@
 import { index, onchainTable, primaryKey } from "@ponder/core";
 
-export const account = onchainTable("account", (t) => ({
-  address: t.hex().primaryKey(),
-  balance: t.bigint().notNull(),
-  isOwner: t.boolean().notNull(),
+export const account = onchainTable("account", (p) => ({
+  address: p.hex().primaryKey(),
```

Comment:
Toss up, but I'm leaning towards using the Drizzle norm of `t` for this. I wrote the new docs this way.